### DISCUSSION TITLE
Inquiry on Next Release and Thread 1.4 Support

### DISCUSSION THREAD

**Discussion by testter2** [openthread/openthread#10561] [Q&A] [answered]
Hi,

I have 3 questions regarding the upcoming openthread release and support for Thread 1.4:

1 - Is there any rough estimate for the next OpenThread release? 

2 - Is there any information available on when OpenThread will support Thread 1.4?  

3- Does following threads cover all open issues related to Thread 1.4 ?
[Thread JIRAs/changes and OT implementation](https://github.com/openthread/openthread/issues/7460)
[[Thread 1.3.x / 1.4] related OT PRs and Thread JIRAs](https://github.com/openthread/openthread/issues/9930).
[Thread Spec Implementation Project page](https://github.com/orgs/openthread/projects/1).

Thanks in advance for your help.

**Comment by jwhui** [comment, marked as answer]
OpenThread Reference Releases tag the actual commit used within the Thread Test Harness. We will tag a new release when the Thread Test Harness is available for Thread 1.4. If you are a member of the Thread Group, you can follow the current Thread Test Harness development efforts.

The GitHub issues you linked identify the tasks related to Thread 1.4.
